src/admin-views/attendees/attendees-event/summary.php

```diff
@@ -103,31 +104,7 @@
 				</ul>
 				<?php do_action( 'tribe_events_tickets_attendees_ticket_sales_bottom', $event_id ); ?>
 			</div>
-			<div class="welcome-panel-column welcome-panel-last alternate">
-				<?php
-				/**
-				 * Fires before the main body of attendee totals are rendered.
-				 *
-				 * @param int $event_id
-				 */
-				do_action( 'tribe_events_tickets_attendees_totals_top', $event_id );
-
-				/**
-				 * Trigger for the creation of attendee totals within the attendee
-				 * screen summary box.
-				 *
-				 * @param int $event_id
-				 */
-				do_action( 'tribe_tickets_attendees_totals', $event_id );
-
-				/**
-				 * Fires after the main body of attendee totals are rendered.
-				 *
-				 * @param int $event_id
-				 */
-				do_action( 'tribe_events_tickets_attendees_totals_bottom', $event_id );
-				?>
-			</div>
+			<?php $this->template( 'attendees/attendees-event/attendance', [ 'event_id' => $event_id ] ); ?>
```

Comment:
I believe it shouldn't be necessary to send the `event_id`.

Reply:
You're right. It's global.